Scripts/Items/Decorative/MonsterStatuette.cs

```diff
@@ -184,6 +171,8 @@ public class MonsterStatuette : Item, IRewardItem
         private MonsterStatuetteType m_Type;
         private bool m_TurnedOn;
         private bool m_IsRewardItem;
+        private Timer m_NewsTimer;
```

Comment:
So, if we're using the old Sherry the Mouse script, while are we adding Town Crier functionality to MonsterStatuette?